### PR TITLE
Update README for Polish localization

### PR DESCRIPTION
**Pull request by mfilocha**
Update README file for Polish localization.

### CODE REVIEW

**Comment by mfilocha** [comment]
Adding @kpucynski and @nvtkaszpir as Polish localization representatives.

File: README-pl.md [hunk 1]
```diff
@@ -29,17 +29,24 @@ cd website
...
-```bash
-# pull in the Docsy submodule
+### Windows
+```powershell
+# aktualizuj podrzędne moduły
 git submodule update --init --recursive --depth 1
 ```
 
+### Linux / inne systemy Unix
+```bash
```
Comment (nvtkaszpir) [inline README-pl.md:39]: do we support generic ````shell` ?

File: README-pl.md [hunk 2]
```diff
@@ -53,11 +60,16 @@ Upewnij się, że zainstalowałeś odpowiednią wersję Hugo "extended", określ
 
 Aby uruchomić i przetestować serwis lokalnie, wykonaj:
 
-```bash
-# install dependencies
-npm ci
-make serve
-```
+- macOS i Linux
+  ```bash
```
Comment (nvtkaszpir) [inline README-pl.md:64]: since few years bash is no longer default shell under Mac OS X

File: README-pl.md (hunk 1 above)
Comment (mfilocha) [reply README-pl.md:39]: I'm following the upstream English version. And I'm okay with this case, it's just a hint for markdown code formatting.

File: README-pl.md (hunk 2 above)
Comment (mfilocha) [reply README-pl.md:64]: See my comment above.

**Review by nvtkaszpir — APPROVED** [review]
/lgtm
/approve

**Comment by mfilocha** [comment]
@sftim Is that expected behaviour?  _"Merging is blocked by issue 44306."_

And what is the current flow for this particular file (localized README). Are English version maintainers only allowed to approve?

**Review by kpucynski — APPROVED** [review]

**Comment by kbhawkey** [comment]
@mfilocha , Can you send a commit to restart the build (probably this pull request was held temporarily for the release).

**Review by Thompson1985 — APPROVED** [review]

**Pull request merged**